### ISSUE TITLE
Can't set platform target with cake 0.5.1 (nuget latest)

### ISSUE THREAD

**Issue by garethbudden** [cake-build/cake#353]
Hi, I've just started playing with cake by using the example repo, and I've updated the cake exe to 0.5.1, the latest on nuget.  
When I try to change the platform target, as shown below, it doesn't seem to pass through the platform target to msbuild and instead still compiles with the default.

```c#
Task("Build")
    .IsDependentOn("Restore-NuGet-Packages")
    .Does(() =>
{
    MSBuild("./src/Example.sln", settings =>
        settings.SetConfiguration(configuration).SetPlatformTarget(PlatformTarget.x64));
});
```

I've looked in the source code and it looks like the _develop_ branch is working correctly but the _main_ isn't.  Is there somewhere to get a nightly build of develop branch?  Or what's the release schedule you try to maintain?

Thanks.

**Comment by patriksvensson** [comment]
Hello @garethbudden,

Thanks for the heads up! This is a known issue that was fixed very recently in #333.
This will appear in the next release (v0.5.2), probably tomorrow or on saturday.

**Comment by garethbudden** [comment]
Ok thanks, just for my info (and possibly others), is there any sort of nightly build for cake?

**Comment by devlead** [comment]
Yes, there's a myget feed at https://www.myget.org/F/cake

**Comment by patriksvensson** [comment]
@garethbudden Yes, we publish CI builds to MyGet: https://www.myget.org/F/Cake/

Until the change is published, you can set the MSBuild property directly:

```csharp
Task("Build")
    .IsDependentOn("Restore-NuGet-Packages")
    .Does(() =>
{
    MSBuild("./src/Example.sln", settings =>
        settings.SetConfiguration(configuration).WithProperty("PlatformTarget", "x86"));
});
```

**Comment by patriksvensson** [comment]
Resolved in v0.5.2.
